requirements/app/base.txt

```diff
@@ -4,7 +4,7 @@ lightning-cloud==0.5.0
 packaging
 deepdiff >= 5.7.0
 starsessions
-fsspec==2022.01.0
+fsspec==2022.7.0
```

Comment:
Is it expected to have a fixed version here rather than setting an upper/lower bound?
```suggestion
fsspec<=2022.7.0
```